### DISCUSSION TITLE
PHY testvector set

### DISCUSSION THREAD

**Discussion by freicyvillalva** [srsran/srsRAN_Project#110] [Ideas & Suggestions]
Hi,
I am installing srsRAN_Project following the manual installation:
https://docs.srsran.com/projects/project/en/latest/user_manuals/source/installation.html#manual-installation
When clicking on the link for the download:
https://github.com/srsran/srsRAN_Project/releases/phy_testvectors.tar.gz
I was getting an error page not found, hence, I advise to change the current link to:
https://github.com/srsran/srsRAN_Project/releases

Bye,
F.

**Comment by andrepuschmann** [comment]
Thanks for flagging this @freicyvillalva . We'll fix that soon

**Comment by brendan-mcauliffe** [comment]
Hey @freicyvillalva this was fixed in an update to the docs today. Thanks for flagging!
